app/controllers/attachments_controller.rb

```diff
@@ -1,15 +1,46 @@
 class AttachmentsController < ApplicationController
   api :GET, '/attachments', 'Return all attachments'
   def index
-    @attachments = Attachment.where(
-      'created_at >= :days and candidate_id == :candidate_id',
-      days: Time.zone.now - 30.days,
-      candidate_id: params[:candidate_id]
-    )
+    if logged(params) == true
```

Comment:
this should be done with a before callback. we want to check for auth for all requests (expected GET /api/jobs)